smithy-model/src/main/java/software/amazon/smithy/model/validation/NodeValidationVisitor.java

```diff
@@ -109,8 +107,17 @@ public enum Feature {
          */
         RANGE_TRAIT_ZERO_VALUE_WARNING,
 
-        // Lowers severity of constraint trait validations to WARNING.
-        ALLOW_CONSTRAINT_ERRORS;
+        /**
+         * Lowers severity of constraint trait validations to WARNING.
+         */
+        ALLOW_CONSTRAINT_ERRORS,
+
+        /**
+         * This setting allows null values to be provided for an optional structure member.
```

Comment:
```suggestion
         * Allows null values to be provided for an optional structure member.
```
nit: match other doc strings